### DISCUSSION TITLE
Forbidden

### DISCUSSION THREAD

**Discussion by thedinz** [RandomNinjaAtk/arr-scripts#130] [Q&A] [answered]
Things were going fine for a while but now I can't get anything, it all says forbidden 403, not sure why, not behind a VPN or blocklist or anything that would cause this.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Sounds like an upstream issue of some sort... Not something that can be fixed here without more context or providing a way to reproduce the issue...
